testsuite/features/support/remote_node.rb

```diff
@@ -266,7 +271,7 @@ def folder_delete(folder)
   #
   # @return [Boolean] true if the node is offline, false otherwise.
   def node_offline?
-    result = run_local('echo test', timeout: 0, check_errors: false).first
```

Comment:
Why did you remove this timeout: 0 ? I suppose it's to make the call instant.

Reply:
On net-ssh if we set it to zero, it gets stuck, I could not find the proper reason for this, but I suspect the timeout is not applied on the same case, in net-ssh it's described as "how long to wait for the initial connection to be made".